tifeatures/dbmodel.py

```diff
@@ -116,21 +116,15 @@ def id_column_info(self) -> Column:  # type: ignore
 
     def columns(self, properties: Optional[List[str]] = None) -> List[str]:
         """Return table columns optionally filtered to only include columns from properties."""
-        cols = [c.name for c in self.properties]
-        if properties is not None:
-            if self.id_column and self.id_column not in properties:
-                properties.append(self.id_column)
-
-            geom_col = self.get_geometry_column()
-            if geom_col:
-                properties.append(geom_col.name)
-
-            cols = [col for col in cols if col in properties]
-
-        if len(cols) < 1:
-            raise TypeError("No columns selected")
-
-        return cols
+        print(f"properties: {properties}")
```

Comment:
```suggestion
```

Reply:
You are too damn fast. I already got rid of it.